Src/BootCamp.Chapter/Calculator/Bmi.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace BootCamp.Chapter.Calculator
+{
+    public class BmiCalculator
```

Comment:
For a class that has no state, that just takes input and returns output, it's perfectly fine to have it as static.